int16 NRays
float64 angleRes
float64 angMax
float64 angMin
float64[] ranges
